roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -13,6 +13,9 @@ storage:
     accesskey: {{ openshift.hosted.registry.storage.s3.accesskey }}
     secretkey: {{ openshift.hosted.registry.storage.s3.secretkey }}
     region: {{ openshift.hosted.registry.storage.s3.region }}
+{%   if openshift.hosted.registry.storage.s3.regionendpoint %}
```

Comment:
Thanks for the hint, that was that I was looking for... Code is updated and pushed.